### PR TITLE
Add Several Configuration in `.github` Folder & Change Response Header Name

### PR DESCRIPTION
**Pull request by arifai**
1. Add `PULL_REQUEST_TEMPLATE`
2. Add Dependabot configuration
3. Change name `errormessage` to `errors`

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    target-branch: develop
+    open-pull-requests-limit: 10
+    commit-message:
+      prefix: bump
+      prefix-development: bump
+      include: scope
+    schedule:
+      interval: weekly
+      day: monday
+      time: 09:00
```
Comment (arifai) [inline .github/dependabot.yml:14]: `time` must be a string not an integer.

**Review by arifai — COMMENTED** [review]
The Dependabot error, please fix this.

**Review by awalnur — APPROVED** [review]
LGTM

File: .github/dependabot.yml (hunk 1 above)
Comment (arifai) [reply .github/dependabot.yml:14]: @awalnur

**Pull request merged**